crates/node/src/lib.rs

```diff
@@ -1132,82 +1077,24 @@ impl Node {
-                    let task = async {
-                        let hash = Hash::hash_json(&transaction)?;
-
-                        self.execute_transaction(&context, transaction, hash).await
-                    };
-
-                    drop(request.outcome_sender.send(task.await.map_err(|err| {
-                        error!(%err, "failed to execute local transaction");
+            )
+            .await
+        };
 
-                        CallError::Mutate(MutateCallError::InternalError)
-                    })));
-                }
-                Finality::Global => {
-                    let (inner_outcome_sender, inner_outcome_receiver) = oneshot::channel();
-
-                    if let Err(err) = self
-                        .call_mutate(&context, transaction, inner_outcome_sender)
-                        .await
-                    {
-                        drop(request.outcome_sender.send(Err(CallError::Mutate(err))));
-                        return;
-                    }
+        drop(request.outcome_sender.send(task.await.map_err(|err| {
+            error!(%err, "failed to execute local action");
 
-                    drop(spawn(async move {
-                        match inner_outcome_receiver.await {
-                            Ok(outcome) => match outcome {
-                                Ok(outcome) => {
-                                    drop(request.outcome_sender.send(Ok(outcome)));
-                                }
-                                Err(err) => {
-                                    drop(request.outcome_sender.send(Err(CallError::Mutate(err))));
-                                }
-                            },
-                            Err(err) => {
-                                error!("Failed to receive inner outcome of a transaction: {}", err);
-                                drop(
-                                    request.outcome_sender.send(Err(CallError::Mutate(
-                                        MutateCallError::InternalError,
-                                    ))),
-                                );
-                            }
-                        }
-                    }));
-                }
-            }
-        } else {
-            match self
-                .call_query(
-                    &context,
-                    request.method,
-                    request.payload,
-                    request.executor_public_key,
-                )
-                .await
-            {
-                Ok(outcome) => {
-                    drop(request.outcome_sender.send(Ok(outcome)));
-                }
-                Err(err) => {
-                    drop(request.outcome_sender.send(Err(CallError::Query(err))));
-                }
-            };
-        }
+            CallError::Mutate(MutateCallError::InternalError)
+        })));
     }
 
+    // TODO: Should we be using this somewhere in the CRDT approach, or not?
```

Comment:
@miraclx just want to make sure this wasn't missed.

Reply:
Seems resolved